recipes/zlib/all/conandata.yml

```diff
@@ -1,11 +1,16 @@
 sources:
+  "1.2.13":
+    url: "https://zlib.net/fossils/zlib-1.2.13.tar.gz"
+    sha256: "b3a24de97a8fdbc835b9833169501030b8977031bcb54b3b3ac13740f846ab30"
   "1.2.12":
     url: "https://zlib.net/fossils/zlib-1.2.12.tar.gz"
     sha256: "91844808532e5ce316b3c010929493c0244f3d37593afd6de04f71821d5136d9"
   "1.2.11":
     url: "https://zlib.net/fossils/zlib-1.2.11.tar.gz"
     sha256: "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1"
 patches:
+  "1.2.13":
+    - patch_file: "patches/0002-gzguts-xcode12-compile-fix.patch"
   "1.2.12":
     - patch_file: "patches/0001-fix-cmake.patch"
```

Comment:
you likely need this patch as well in 1.2.13

Reply:
This one didn't apply cleanly anymore and the build was fine locally... Let's hope that we don't need it anymore and it's fixed in zlib :-)

Reply:
oh... I think you're indeed right...

Reply:
Rebased and fixed